extensions/transfer/transfer-core/src/main/java/com/microsoft/dagx/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -148,6 +150,19 @@ private void run() {
         }
     }
 
+
+    private int checkDeprovisioned() {
```

Comment:
Do I read this correct: The state is just incremented without anything happening?